Does not install with NLS disabled

**Describe the bug**
When configured with `-DOPTION_BUILD_TRANSLATIONS=OFF`, install target fails:

```
CMake Error at cmake_install.cmake:151 (file):
  file INSTALL cannot find
  "/wrkdirs/usr/ports/games/widelands/work/.build/locale": No such file or
  directory.
```
while with the enabled option it's:
```
-- Installing: /wrkdirs/usr/ports/games/widelands/work/stage/usr/local/share/widelands/locale
-- Installing: /wrkdirs/usr/ports/games/widelands/work/stage/usr/local/share/widelands/locale/el
-- Installing: /wrkdirs/usr/ports/games/widelands/work/stage/usr/local/share/widelands/locale/el/LC_MESSAGES
-- Installing: /wrkdirs/usr/ports/games/widelands/work/stage/usr/local/share/widelands/locale/el/LC_MESSAGES/maps.mo
-- Installing: /wrkdirs/usr/ports/games/widelands/work/stage/usr/local/share/widelands/locale/el/LC_MESSAGES/scenario_tutorial03_seafaring.wmf.mo
-- Installing: /wrkdirs/usr/ports/games/widelands/work/stage/usr/local/share/widelands/locale/el/LC_MESSAGES/world.mo
```
it looks like it tries to install files which are not generated.

Additionally, it seems to require Gettext regardless of the option setting.

**Expected behavior**
Installation should pass.

**Version:**
 - OS: FreeBSD 13.0
 - Widelands Version: 1.0

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -548,15 +548,16 @@ install(
   PATTERN "CMakeLists.txt" EXCLUDE
 )
 
-set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES ${WL_INSTALL_DATADIR}/locale)
-
-install(
-  DIRECTORY
-    ${CMAKE_CURRENT_BINARY_DIR}/locale/
-  DESTINATION ${WL_INSTALL_DATADIR}/locale
-  CONFIGURATIONS Debug;Release;None
-  COMPONENT CoreLanguageFiles
-)
+if(OPTION_BUILD_TRANSLATIONS)
+  set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES ${WL_INSTALL_DATADIR}/locale)
+  install(
+    DIRECTORY
+      ${CMAKE_CURRENT_BINARY_DIR}/locale/
+    DESTINATION ${WL_INSTALL_DATADIR}/locale
+    CONFIGURATIONS Debug;Release;None
+    COMPONENT CoreLanguageFiles
+  )
+endif(OPTION_BUILD_TRANSLATIONS)
 
 if (UNIX AND NOT APPLE)
   add_subdirectory(xdg)

